src/sdk/sql_cluster_router.cc

```diff
@@ -3131,6 +3131,13 @@ hybridse::sdk::Status SQLClusterRouter::HandleLongWindows(
         if (!extract_status.IsOK()) {
             return extract_status;
         }
+        std::set<std::string> distinct_long_window;
+        for (const auto& info : long_window_infos) {
+            distinct_long_window.insert(info.window_name_);
+        }
+        if (distinct_long_window.size() != long_window_map.size()) {
+            return {base::ReturnCode::kError, "long_windows option doesn't match window in sql"};
+        }
```

Comment:
not exactly like this.

we want to check any `window_name` declared in the `long_windows` options should exist in the window definitions.